tests/cases/compiler/moduleCrashInJSFile.ts

```diff
@@ -0,0 +1,14 @@
+// @filename: a.js
+// @checkJs: true
+// @outDir: out/
+
+//// [thisKeyword.ts]
+module foo {
+    this.bar = 4;
+}
+
+//// [thisKeyword.js]
+var foo;
+(function (foo) {
+    this.bar = 4;
+})(foo || (foo = {}));
```

Comment:
This is part of the .js emit for the `thisKeyword` test and shouldn't be part of the test case.